src/tools/daxio/daxio.c

```diff
@@ -123,31 +139,23 @@ print_version(void)
 static void
 print_usage(void)
 {
-	printf("Usage: daxio [option] ...\n");
-	printf("Valid options:\n");
-	printf("-i, --input=FILE  - input device/file (default stdin)\n");
-	printf("-o, --output=FILE - output device/file (default stdout)\n");
-	printf("-k, --skip=BYTES  - skip offset for input (default 0)\n");
-	printf("-s, --seek=BYTES  - seek offset for output (default 0)\n");
-	printf("-l, --len=BYTES   - total length to perform the I/O\n");
-	printf("-z, --zero        - zeroing the device\n");
-	printf("-h. --help        - print this help\n");
-	printf("-V, --version     - display version of daxio\n");
+	printf(USAGE_MESSAGE);
 }
 
 /*
  * long_options -- command line options
  */
 static const struct option long_options[] = {
-	{"input",	required_argument,	NULL,	'i'},
-	{"output",	required_argument,	NULL,	'o'},
-	{"skip",	required_argument,	NULL,	'k'},
-	{"seek",	required_argument,	NULL,	's'},
-	{"len",		required_argument,	NULL,	'l'},
-	{"zero",	no_argument,		NULL,	'z'},
-	{"help",	no_argument,		NULL,	'h'},
-	{"version",	no_argument,		NULL,	'V'},
-	{NULL,		0,			NULL,	 0 },
+	{"input",			required_argument,	NULL,	'i'},
+	{"output",			required_argument,	NULL,	'o'},
+	{"skip",			required_argument,	NULL,	'k'},
+	{"seek",			required_argument,	NULL,	's'},
+	{"len",				required_argument,	NULL,	'l'},
+	{"zero",			no_argument,		NULL,	'z'},
+	{"do-not-clear-bad-blocks",	no_argument,		NULL,	'n'},
```

Comment:
1) I think this option should have an argument that says whether bad blocks are enabled or not.
2) "b" switch would make more sense